src/store/blocks/getters.ts

```diff
@@ -6,25 +8,25 @@ import { State as RootState, RootStore } from '../state';
 const { read } = getStoreAccessors<BlocksState, RootState>('blocks');
 
 const getters = {
-  blocksById: (state: BlocksState): { [id: string]: Block } => state.byId,
+  blocks: (state: BlocksState): { [id: string]: Block } => state.blocks,
 
   blockIds(state: BlocksState): string[] {
-    return state.allIds;
+    return map(state.blocks, (_: Block, key: string) => key);
   },
 
   allBlocks(state: BlocksState): Block[] {
-    return state.allIds.map(id => state.byId[id]);
+    return map(state.blocks, (block: Block) => block);
```

Comment:
Is this case for instance, I'd prefer to use `Object.keys(state.blocks).map(key => state.blocks[key])` since it's standard JS of using `map`. lodash's map also supports mapping over object keys which in my eyes makes it a bit confusing.